usr/src/cmd/dumpadm/svc-dumpadm

```diff
@@ -147,11 +153,14 @@ if [ "x$DUMPADM_ENABLE" != xno ]; then
 		    mksavedir && /usr/bin/savecore $DUMPADM_SAVDIR &
 		fi
 	else
+		[[ -f "$DUMPADM_SAVDIR/keyfile" ]] && \
+			keyarg="-k $DUMPADM_SAVDIR/keyfile"
+
 		#
 		# The dump device couldn't have been dedicated before we
 		# ran dumpadm, so we must execute savecore again.
 		#
-		mksavedir && /usr/bin/savecore $DUMPADM_SAVDIR &
+		mksavedir && /usr/bin/savecore $keyarg $DUMPADM_SAVDIR &
```

Comment:
Undefined "keyarg" will be a null string, right?

Reply:
Probably best to initialize it to make that explicit.